src/main/java/pico/typecheck/PICOVisitor.java

```diff
@@ -375,30 +375,6 @@ private void checkNewInstanceCreation(Tree node) {
         }
     }
 
-    @Override
-    public Void visitMethodInvocation(MethodInvocationTree node, Void p) {
-        super.visitMethodInvocation(node, p);
-        ParameterizedExecutableType mfuPair =
-                atypeFactory.methodFromUse(node);
-        AnnotatedExecutableType invokedMethod = mfuPair.executableType;
-        ExecutableElement invokedMethodElement = invokedMethod.getElement();
-        // Only check invocability if it's super call, as non-super call is already checked
-        // by super implementation(of course in both cases, invocability is not checked when
-        // invoking static methods)
-        if (!ElementUtils.isStatic(invokedMethodElement) && TreeUtils.isSuperCall(node)) {
-            checkMethodInvocability(invokedMethod, node);
-        }
-        return null;
-    }
-
-    private void saveFbcViolatedMethods(ExecutableElement method, String actualReceiver, String declaredReceiver) {
-        if (actualReceiver.contains("@UnderInitialization") && declaredReceiver.contains("@Initialized")) {
-            String key = ElementUtils.enclosingClass(method) + "#" + method;
-            Integer times = fbcViolatedMethods.get(key) == null ? 1 : fbcViolatedMethods.get(key) + 1;
-            fbcViolatedMethods.put(key, times);
```

Comment:
Can you also remove fields `fbcViolatedMethods` and `shouldOutputFbcError`?

Reply:
These fields are used in `PICOChecker`, should I also removed the corresponding methods, e.g., `PICOChecker.printFbcViolatedMethods()`?

Reply:
Is that method still being called? I didn't see other uses in this class, but didn't check other classes. Maybe the field is in the wrong class if it is only used from another class?

Reply:
If we pass `-AprintFbcErrors` as the argument, `PICOChecker.printFbcViolatedMethods()` will try to print the Fbc Violations, e.g., 
```
===================== Fbc Violations Found! ======================
                        Method                Violated Times
typecheck.FbcViolatingMethod#foo()                             1
====================================================================
```
If we don't want this feature, it is safe to remove all the related code.

---

The two fields `fbcViolatedMethods` and `shouldOutputFbcError` and the method `saveFbcViolatedMethods` were used in `PICOVisitor.checkMethodInvocability()`, which was removed by baorui because the check is unnecessary. The corresponding [link](https://github.com/opprop/immutability/commit/9e279221d1fbb7953230db67f60053f73355a56d#diff-26f676b2439cdaab28cdc4ec78efb52fL436).

So, I think these fields used to be in the right place. Now, it depends on whether we want to show this specific output.

Reply:
Does the counters still get incremented somewhere? If so, I'm fine with keeping the feature.
However, if this was the only place that counted violations, we should completely remove the cruft.

Reply:
The counter is only get incremented in `checkMethodInvocability()`. I will remove all the related code.